vernac/comProgramFixpoint.ml

```diff
@@ -117,7 +117,12 @@ let nf_evar_context sigma ctx =
 let build_wellfounded pm (recname,pl,bl,arityc,body) poly ?typing_flags ?deprecation ?using r measure notation =
   let open EConstr in
   let open Vars in
-  Coqlib.check_required_library ["Coq";"Program";"Wf"];
+  let fix_sub_ref, measure_on_R_ref = try fix_sub_ref (), measure_on_R_ref ()
+    with NotFoundRef r ->
+      CErrors.user_err
+        Pp.(str r ++ spc() ++ str "not registered," ++ spc() ++
+            str "library Coq.Program.Wf has to be required first.")
```

Comment:
The error is a bit too radical, it should rather say something like "you should try requiring library Coq.Program.Wf". Indeed there are other ways to register the missing values.

Reply:
changed